web/styles/buttons.css

```diff
@@ -354,3 +354,199 @@
         );
     }
 }
+
+.icon-button {
+    display: flex;
+    font-size: var(--base-font-size-px);
+    color: var(--color-text-neutral-icon-button);
+    background-color: transparent; /* Override button default background color */
+    padding: 0.375em;
+    cursor: pointer;
+    border: none;
+    border-radius: 4px;
+
+    &:hover {
+        color: var(--color-text-neutral-icon-button-hover);
+    }
+
+    &:active {
+        color: var(--color-text-neutral-icon-button-active);
+    }
+
+    &.icon-button-square {
+        &:hover {
+            background-color: var(
+                --color-background-neutral-icon-button-square-hover
+            );
+        }
+
+        &:active {
+            color: var(--color-text-neutral-icon-button-square-active);
+            background-color: var(
+                --color-background-neutral-icon-button-square-active
+            );
+        }
+    }
+}
```

Comment:
Does this class need these color definitions? If it's supposed to always be used with one of the overriding classes, it seems better to not declare any colors here at all, and just have a comment noting that the colors will be declared in one of the components extending this.

Reply:
This seems to work:
``` diff
diff --git a/web/styles/buttons.css b/web/styles/buttons.css
index b31c5333e7..6dea7a8dbd 100644
--- a/web/styles/buttons.css
+++ b/web/styles/buttons.css
@@ -356,37 +356,15 @@
 }
 
 .icon-button {
+    /* This class should always be used with an icon-button-* class
+       defining the color scheme of the button, defined below. */
     display: flex;
     font-size: var(--base-font-size-px);
-    color: var(--color-text-neutral-icon-button);
     background-color: transparent; /* Override button default background color */
     padding: 0.375em;
     cursor: pointer;
     border: none;
     border-radius: 4px;
-
-    &:hover {
-        color: var(--color-text-neutral-icon-button-hover);
-    }
-
-    &:active {
-        color: var(--color-text-neutral-icon-button-active);
-    }
-
-    &.icon-button-square {
-        &:hover {
-            background-color: var(
-                --color-background-neutral-icon-button-square-hover
-            );
-        }
-
-        &:active {
-            color: var(--color-text-neutral-icon-button-square-active);
-            background-color: var(
-                --color-background-neutral-icon-button-square-active
-            );
-        }
-    }
 }
 
 .icon-button-neutral {
```